transaction-service/src/main/kotlin/ua/com/foxminded/restClient/mapper/ProtoMapper.kt

```diff
@@ -0,0 +1,28 @@
+package ua.com.foxminded.restClient.mapper
+
+import com.google.protobuf.Timestamp
+import java.time.ZoneOffset
+import org.springframework.stereotype.Component
+import proto.ProtoMessage
+import ua.com.foxminded.restClient.dto.TransactionDto
+
+@Component
```

Comment:
why this class should be a bean?

Reply:
okay. I Added it to my transaction mapper